terra-batch-libchart/values.yaml

```diff
@@ -2,7 +2,7 @@ exports:
   common:
     cromwell:
       name: coa-cromwell-svc
-      image: broadinstitute/cromwell:89-561bbf4
+      image: broadinstitute/cromwell:latest
```

Comment:
Would be nice to add a comment here describing what the `latest` tag is, since we're doing something a little non-standard for Terra services.